src/styles/app.scss

```diff
@@ -5,3 +5,34 @@ code {
   padding: 0.6rem;
   background-color: $nhsuk-border-color;
 }
+
+.notify-content {
+  background-color: $color_nhsuk-white;
+  border: 1px solid $nhsuk-border-color;
+}
+
+.notify-cognito-login-form {
+  [data-amplify-authenticator] {
+    [data-amplify-container] {
```

Comment:
```suggestion
.notify-cognito-login-form {
  margin-top: 1.5rem;
  [data-amplify-authenticator] {
    [data-amplify-container] {
```
I'm not sure if this is the best way to add a gap, but a gap looks much better to me.

Reply:
![image](https://github.com/user-attachments/assets/71b03a6b-74ed-49be-ad3e-e4f5d40d5abc)